src/txFactory/identityManagerTransactionFactory.ts

```diff
@@ -327,10 +338,11 @@ export function setUrlAOA(web3, didEntity, urlAOA) {
 export function getEntity(web3, didEntity) {
   const entityAddr = AIdUtils.getProxyAddress(didEntity)
   const transaction = Object.assign({}, config.basicTransaction)
-  transaction.data = web3.eth.abi.encodeFunctionCall(
+  const delegatedData = web3.eth.abi.encodeFunctionCall(
     config.contractsAbi.AlastriaIdentityEntity.getEntity,
     [entityAddr]
   )
+  transaction.data = delegated(web3, delegatedData)
```

Comment:
In a call function (getters) there is no need to use the proxy. Revert this function.